### DISCUSSION TITLE
Exception stack trace is sometimes truncated to a single entry

### DISCUSSION THREAD

**Discussion by NameOfTheDragon** [datalust/seq-tickets#2179] [Help]
Why is it that sometimes my exception stack traces are truncated to a single entry? Something must have called the method so why isn't that included in the stack trace? I can't figure out why this is happening, can you shed any light on it? I'm logging from C# via NLog. I'm not sure whether this is happening in NLog, the NLog Seq target, or in Seq itself.

![image](https://github.com/datalust/seq-tickets/assets/1268490/473f3723-e9cf-471c-b921-b7ad90046e10)

**Comment by nblumhardt** [comment]
Hi @NameOfTheDragon,

If you click on the "Export" menu and "Download raw JSON", is there more exception detail included? (This will pinpoint it to either the Seq UI, or the client side, I think.)

Best regards,
Nick
